File: app/src/main/java/tbsc/muzz/MuzzApplication.java
package tbsc.muzz;

import android.app.Application;

/**
 * @author tbsc on 26/03/2018
 */
public class MuzzApplication extends Application {

    private static MuzzApplication instance;

    public static MuzzApplication get() {
        return instance;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }

}

File: app/src/main/java/tbsc/muzz/MuzzApplication.kt
package tbsc.muzz

import android.app.Application

/**
 * @author tbsc on 26/03/2018
 */
class MuzzApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this
    }

    companion object {

        private var instance: MuzzApplication? = null

        fun get(): MuzzApplication {
            return instance!!
        }
    }

}
